release/_releases_json.sh

```diff
@@ -31,6 +47,28 @@ function _merge_json_snippets {
 	fi
 }
 
+function _process_new_product {
+	local product_name="${1}"
+
+	local downloaded_releases_json="${_RELEASE_ROOT_DIR}/release-data/promotion/files/0000-00-00-releases.json"
```

Comment:
@Wykenz, let's keep the consistency, e.g. let's rename the variable to `releases_json` only since we renamed the file name. :slightly_smiling_face: